packages/profiling-node/bindings/cpu_profiler.cc

```diff
@@ -604,8 +629,15 @@ static void GetSamples(const napi_env &env, const v8::CpuProfile *profile,
       }
     }
 
-    napi_value sample = CreateSample(
-        env, stack_index, sample_timestamp - profile_start_time_us, thread_id);
+    uint64_t sample_delta_us = sample_timestamp - profile_start_time_us;
+    uint64_t sample_timestamp_ns = sample_delta_us * 1e3;
```

Comment:
l: can we create helpers for timestamps here. Feels like it's very easy to get lose with an exponent.

Reply:
I changed to naming and flow so it's more clear what it is doing. I will create a helper for the other conversion